hal/armv7a/imx6ull/_init.S

```diff
@@ -86,33 +345,6 @@ _cpy4:
 _start:
 	cpsid aif, #SYS_MODE
 
-	ldr r0, =relOffs
-	ldr r1, =#VADDR_SYSPAGE
-	lsr r1, #12
-	lsl r1, #12
-
-	sub r0, r0, #VADDR_KERNEL
-	add r0, r0, #ADDR_DDR
-	sub r2, r1, r9
-	str r2, [r0]
-
-	ldr r0, [r9, #4] /* syspage size address - syspage address + sizeof(hal_syspage_t) */
-	add r2, r9, r0   /* end of the syspage */
-
-	ldr r0, =syspage
-	sub r0, r0, #VADDR_KERNEL
-	add r0, r0, #ADDR_DDR
-	str r1, [r0]
-
-	sub r1, r1, #VADDR_KERNEL
-	add r1, r1, #ADDR_DDR
-
-syspage_cpy:
-	ldr r3, [r9], #4
-	str r3, [r1], #4
-	cmp r9, r2
-	blo syspage_cpy
-
```

Comment:
Hmmm, we don't need this? Where are syspage programs unpacked?

Reply:
In new approach without plo this code will not be used. The programs unpacking will be introduced in the next PR, this one includes only restoring previous code.

Reply:
Hmm, maybe introduce all of it in the same PR but separate commits? This will still be the same topic de facto, so it can be in the same PR.

Reply:
This PR restores the previous code in `_init.S`. The code related to copying programs and making modification in syspage in not related to it and I wanted to make it readable for reviewer. 

It's up to You, You are reviewer. I can put the additional commit within this PR.